packages/experimental/MenuButton/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui-react-native/experimental-menu-button",
-  "version": "0.5.24",
+  "version": "0.5.25",
```

Comment:
NPM says last published is 0.5.24

https://www.npmjs.com/package/@fluentui-react-native/experimental-menu-button